test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
+Feature: Version command
+Command "minishift version" shows version of minishift binary.
+
+  Scenario: Setup-cdk installs and enables specific addons
+      When executing "minishift version" succeeds
+      Then stdout should match "^minishift v\d\.\d\.\d\+[\w\d]{7}\n$"
```

Comment:
@hferentschik Useful, thanks. However I am not sure why there is `{7-8}`, should not --short flag give seven characters as sha result?